MockDuck/Sources/MockSerializable.swift

```diff
@@ -78,14 +78,26 @@ private extension URLRequest {
 
         var hashData = Data()
 
-        if let urlData = normalizedRequest.url?.absoluteString.data(using: .utf8) {
+        var components = URLComponents(url: normalizedRequest.url!, resolvingAgainstBaseURL: false)
```

Comment:
sorting the query parameters in the URL so that we get the same hash for the same request and can return saved response, even if the order of query parameters is changed.